langlib/langlib-test/src/test/java/org/ballerinalang/langlib/test/statements/foreach/ForeachJSONTests.java

```diff
@@ -94,15 +94,6 @@ public void testJSONBoolean() {
         Assert.assertEquals(returns[0].stringValue(), result);
     }
 
-    @Test(expectedExceptions = BLangRuntimeException.class,
-            expectedExceptionsMessageRegExp = ".*incompatible types: 'error' cannot be cast to 'map<json>'.*")
-    public void testJSONNull() {
```

Comment:
This too we don't need to remove, right?

Reply:
Same as above.